.mockery.yaml

```diff
@@ -0,0 +1,6 @@
+packages:
+  github.com/nephio-project/nephio/controllers/pkg/giteaclient:
```

Comment:
Since, we want to generate mock for the `GiteaClient` interface, the package path to the interface has to be provided. Refer https://vektra.github.io/mockery/latest/features/#packages-configuration as well